CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ BUG FIXES:
 
 ## 1.13.2 (September 20, 2022)
 
+BREAKING CHANGES:
+
+* ca: If using Vault as the service mesh CA provider, the Vault policy used by Consul now requires the `update` capability on the intermediate PKI's tune mount configuration endpoint, such as `/sys/mounts/connect_inter/tune`. This change was necessary to address [[GH-14516](https://github.com/hashicorp/consul/issues/14516)]. Refer to [upgrade guidance](https://www.consul.io/docs/upgrading/upgrade-specific#modify-vault-policy-for-vault-ca-provider) for more information.
```

Comment:
Note to self: document that we're looking into making this better in an upcoming patch